.github/workflows/ci.yml

```diff
@@ -167,7 +167,7 @@ jobs:
           key: ${{ env.opam_cache_key }}
 
       - name: Use OCaml ${{matrix.ocaml_compiler}}
-        uses: ocaml/setup-ocaml@v3.1.4
+        uses: ocaml/setup-ocaml@v3
```

Comment:
Thanks @smorimoto!

But so far I have preferred specifying the full version to reduce the risk of breakage (because of the additional caching we do).

I have also incremented the cache version whenever I upgraded setup-ocaml (maybe being overly cautious here?).

Reply:
OK